python/foglamp/services/core/api/certificate_store.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+import os
+from aiohttp import web
+from foglamp.common import logger
+
+__author__ = "Ashish Jabble"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+_FOGLAMP_DATA = os.getenv("FOGLAMP_DATA", default=None)
+_FOGLAMP_ROOT = os.getenv("FOGLAMP_ROOT", default='/usr/local/foglamp')
+
+
+_logger = logger.setup(__name__, level=20)
+
+_help = """
+    -------------------------------------------------------------------------------
+    | POST             | /foglamp/certificate                                          |
+    -------------------------------------------------------------------------------
+"""
+
+async def upload(request):
+    """
+
+    :Example:
+        curl -F "key=@filename.key" -F "cert=@filename.cert" http://localhost:8081/foglamp/certificate
+
+    """
+    data = await request.post()
+
+    # contains the name of the file in string format
+    key_file = data.get('key')
+    cert_file = data.get('cert')
+
+    # TODO: handle overwrite if file with the same name already exists?
+    allow_overwrite = data.get('overwrite', 0)
+    should_overwrite = True if allow_overwrite == 1 else False
```

Comment:
@MarkRiddoch I think we need overwrite functionality which by default will give bad request if filename with same name already exists.

Reply:
Sure, we probably should also have delete and put for removing certs and for updating them. Will update admin api document

Reply:
I think explicit endpoint for update can be eliminated with ` 'overwrite': 1`